### PR TITLE
For Ember Times No. 111: Add Writeup on Crash Log Podcast

### PR DESCRIPTION
**Pull request by jayjayjpg**

### CODE REVIEW

**Review by Alonski — APPROVED** [review]

File: source/2019-08-16-the-ember-times-issue-111.md [hunk 1]
```diff
@@ -118,15 +118,6 @@ In the latest [episode on the Crash Log Podcast](https://crashlog.transistor.fm/
 <!-- alex ignore crash -->
 Listen to the full episode on [the Crash Log Podcast website](https://crashlog.transistor.fm/episodes/episode-6)!
 
-
```
Comment (amyrlam) [inline source/2019-08-16-the-ember-times-issue-111.md]: @jessica-jordan let's add the rule to ignore, if it was causing issue. what rule was this? https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md012?

Comment (jayjayjpg) [reply source/2019-08-16-the-ember-times-issue-111.md]: Yes, exactly that's the one:

```
fatal: remote base already exists.
source/2019-08-16-the-ember-times-issue-111.md: 121: MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
source/2019-08-16-the-ember-times-issue-111.md: 122: MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 3]
source/2019-08-16-the-ember-times-issue-111.md: 123: MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 4]
The command "bash scripts/markdown-lint.sh" exited with 1.
```

I can update the `markdownlint` config accordingly :+1:

Comment (amyrlam) [reply source/2019-08-16-the-ember-times-issue-111.md]: nice, thanks! sorry it's been a bit wonky, def don't want to introduce blockers

File: source/2019-08-16-the-ember-times-issue-111.md [hunk 2]
```diff
@@ -48,72 +49,75 @@ and last, but not least,
...
+
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+<!-- alex ignore crash -->
+## [Learn about HTML Scopes and Fastboot Complexity with the Crash Log Podcast](https://crashlog.transistor.fm/episodes/episode-6)
```
Comment (amyrlam) [inline source/2019-08-16-the-ember-times-issue-111.md]:
```suggestion
## [Learn about HTML Scopes and Fastboot Complexity with the Crash Log Podcast 🎙](https://crashlog.transistor.fm/episodes/episode-6)
```

**Review by amyrlam — APPROVED** [review]
added one emoji, otherwise reads great! 

thanks for the writeup and for goodbits-ing this week! 🙏

**Pull request merged**